lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,246 @@
+      font-style: italic;
+      text-align: center;
+      &__links {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      &__links__element a {
+        display: inline-block;
+        padding: 0.5rem;
+        color: $lightblue;
+        text-decoration: none;
+      }
+      &__links__element:hover {
+        background-color: $blue;
+      }
+    }
+  }
+  .active {
+    background-color: $darkgray;
+  }
+}
+// responsive sizing for "screen and (min-width: 768px)"
+@include media-query('tab') {
+  .Main {
+    font-family: 'Roboto', sans-serif;
+    display: block;
+    background-color: $black;
+    color: $gray;
+    padding: 1rem;
+    // Header styling
+    &__header {
+      text-align: center;
+      padding: 1rem;
+      &__title {
+        font-family: 'Caveat', cursive;
+        color: $darkred;
+        font-size: 3rem;
+        margin: 0;
+        padding: 0;
+      }
+      &__subtitle {
+        font-family: 'Courgette', cursive;
+        color: $robins;
+        font-size: 2rem;
+        margin-top: 0;
+        margin-bottom: 1rem;
+      }
+      // Nav menu
+      // Animation inspired by https://codepen.io/EvyatarDa/pen/waKXMd
+      // Fix position problems - causing animation not to stick to the li
+      &__menu__list {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      &__menu__list__page {
+        float: left;
+      }
+      &__menu__list__page a {
+        display: block;
+        padding: 0.5rem;
+        color: $lightblue;
+        text-decoration: none;
+      }
+      &__menu__list__page a:before, &__menu__list__page a:after {
+        position: fixed;
+        left: 0px;
+        width: 100%;
+        height: 2px;
+        background: $lightblue;
+        content: "";
+        opacity: 0;
+        transition: all 0.3s;
+      }
+      &__menu__list__page a:before {
+        top: 0px;
+        transform: translateY(10px);
+      }
+      &__menu__list__page a:after {
+        bottom: 0px;
+        transform: translateY(-10px);
+      }
+      &__menu__list__page:hover {
+        background-color: $blue;
+      }
+      &__menu__list__page a:hover:before, &__menu__list__page a:hover:after {
+        opacity: 1;
+        transform: translateY(0px);
+      }
+    }
+    // Gallery styling
+    &__content {
+      &__title {
+        font-family: 'Courgette', cursive;
+        padding: 0 1rem;
+      }
+      &__pgdesc {
+        padding: 0 1rem;
+      }
+      &__gallery {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+      }
+      &__gallery__card {
+        border: 2px solid $darkgray;
+        border-radius: 5px;
+        text-align: center;
+        margin: 1rem;
+        max-width: 45%;
+        transition: 0.5s;
+      }
+      &__gallery__card:hover {
+        border: 2px dashed $purple;
+        transition: 0.5s;
+        transform: scale(1.1);
```

Comment:
Nice work on the CSS animations. Hover animations are a classic example, and you'll typically be able to find at least one use on most websites!